examples/basic/packages/tsconfig/base.json

```diff
@@ -2,20 +2,18 @@
   "$schema": "https://json.schemastore.org/tsconfig",
   "display": "Default",
   "compilerOptions": {
-    "composite": false,
-    "declaration": true,
-    "declarationMap": true,
     "esModuleInterop": true,
-    "forceConsistentCasingInFileNames": true,
-    "inlineSources": false,
-    "isolatedModules": true,
-    "moduleResolution": "node",
-    "noUnusedLocals": false,
-    "noUnusedParameters": false,
-    "preserveWatchOutput": true,
     "skipLibCheck": true,
+    "target": "es2022",
```

Comment:
@mattpocock is this needed? Doesn't `"lib": ["es2022", "dom", "dom.iterable"]` already implies `"target": "es2022"`?

Reply:
Confusingly, the TS team recommends you keep "target" in place. I still haven't quite grokked why - but I think TS always expects there to be a "target" field.